Cargo.toml

```diff
@@ -17,13 +17,14 @@ bevy = { version = "0.12", default-features = false, features = [
     "bevy_sprite",
     "multi-threaded",
     "png",
+    "wayland",
```

Comment:
Sorry, without this it doesn't run on my machine, feel free to remove it of course